src/pages/StartPage/StartPage.tsx

```diff
@@ -1,8 +1,21 @@
 import { useNavigate } from 'react-router-dom';
 import logo_big_shadow from '@/assets/logo_big_shadow.png';
+import { useGuestLogin } from '@/hooks/useAuth';
 
 const StartPage = () => {
   const navigate = useNavigate();
+
+  const { mutate } = useGuestLogin({
```

Comment:
```suggestion
  const { mutate: guestLoginMutate } = useGuestLogin({
```
등 더 명시적인 이름은 어떠신가요 !?!

Reply:
더 명시적이어서 보기 좋은거 같습니다!